### ISSUE TITLE
Allow for setting content warnings on by default as a "subject" instance-wide

### ISSUE THREAD

**Issue by bgcarlisle** [tootsuite/mastodon#7870]
Whenever new people join my instance, there's always a flurry of posts without CW's that really need them. This leads to awkward discussions about community standards with new users about how "you really should have put a CW on that post, because it's off-topic and political," for example.

This turns many users off, and after a few cycles of this, I'm starting to wonder if the way that CW's are designed as being *opt-in* might be prompting this sort of behaviour.

May I suggest that there be an option to set CW's as on by default for all users on an instance?

I don't mean mandatory CW's, but make them more like email subject lines. I imagine a system where users can disable CW's on a per-post basis, but the default would be to have them visible, and maybe call them a "subject" in order to back off the connotation of "content warning" that they are only for topics that are extremely distressing for everyone.

And because I'm sure there will be instance admins who balk at this suggestion, making it a setting that an instance admin can change (opt-in vs opt-out) would be good too. :P

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by jchalifour** [comment]
And if you give people a setting in their preferences to auto-expand CWs (subjects) then each person can choose how they like to scan through their timelines (i.e. seeing all text expanded or just seeing the subject/CW text and expanding when they want to read more).

**Comment by bgcarlisle** [comment]
Is there any possibility that this will be implemented?

**Comment by Hache-Games** [comment]
This is a really good idea and I can't fathom why it's sitting here unaddressed.

**Comment by jordigh** [comment]
> Is there any possibility that this will be implemented?

Yes, it's possible. I'll look into it. Give me a weekend or two to see what I can do.

**Comment by rpcutts** [comment]
Just throwing in a (very possibly minority) dissenting opinion.

Liberal use of CW is my least favourite thing about mastodon. To me, making subject/content the default behaviour for toots would add a lot of timeline clutter that seems out of place for a microblogging timeline.

**Comment by ThisIsMissEm** [comment]
This seems like it may be a duplicate of #3903
